drivers/atlona/server.go

```diff
@@ -38,18 +38,17 @@ func main() {
 	}
 
 	// import driver library
-	createVS := func(ctx context.Context, addr string) drivers.VideoSwitcherDSP {
+	createVS := func(ctx context.Context, addr string) (drivers.VideoSwitcherDSP, error) {
 		deviceType, err := atlona.GetDeviceType(context.TODO(), addr)
 		if err != nil {
-			fmt.Printf("failed to discover device: %s\n", err)
-			os.Exit(1)
+			return nil, fmt.Errorf("failed to discover device: %w", err)
 		}
```

Comment:
Let's add a space after the `if` block :)

Reply:
no